visualization/dashboard/src/MLIDashboard/Controls/CohortList/CohortList.tsx

```diff
@@ -97,9 +117,24 @@ export class CohortList extends React.PureComponent<ICohortListProps, ICohortLis
                         onDismiss={this.onCancel}
```

Comment:
I think we'd discussed moving the existing "FilterEditor" to be "CohortEditor" and should contain all the cohort editor state and styling. That way, this class can just manage the list of cohorts and the buttons to open the cohort editor and show a cohort editor when cohort-to-edit is set. Then the cohortEditor.tsx can maintain the state of cohort.name and cohort.listOfFilters, until the user presses save or closes the dialog.